snippets/csharp/VS_Snippets_CLR/Cryptography.3DES.Create.Memory/CS/memoryexample.cs

```diff
@@ -59,7 +59,7 @@ static byte[] encryptStringToBytes_AesManaged(string plainText, byte[] Key, byte
                 AesManagedAlg.Key = Key;
                 AesManagedAlg.IV = IV;
 
-                // Create a decrytor to perform the stream transform.
+                // Create an encrytor to perform the stream transform.
```

Comment:
Typo: should be *encryptor*

Reply:
Good catch! That’s what I had in mind to do. 😊